AGENTS.md

```
@@ -0,0 +1,46 @@
+corepack enable            # activate the pinned Yarn
+yarn install --immutable   # install dependencies
+yarn build-typechain       # generate src/typechain from src/abis (run before build)
+yarn build                 # rollup bundle -> dist/
+yarn lint                  # tsc --noEmit + eslint
+yarn format                # prettier --write
+yarn test                  # jest — integration tests that hit live contracts over RPC (not run in CI)
+```
+
+## Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)